users/strelaysrv.rst

```diff
@@ -89,6 +89,10 @@ Options
     "https://relays.syncthing.net/endpoint"). Blank to disable announcement to
     a pool, thereby remaining a private relay.
 
+.. cmdoption:: -pprof
+
+    Enable the built in profiling on the status server
+
```

Comment:
Does this mean the `Go` profiler? Would this dump profile data? Or is this the `/status` endpoint - wasn't that enabled by default?

Maybe it's just me but I don't understand this sentence as written :)

Reply:
This was copy-pasted from the `strelaysrv -help` output. It's in the relaysrv source code here: https://github.com/syncthing/syncthing/blob/81a4b22d4342b187ca910a1e6f4ce0ff61b89569/cmd/strelaysrv/main.go#L102

What it does is it enables a new `/debug/pprof` endpoint you can request to get a Go profile (see the code here: https://github.com/syncthing/syncthing/blob/81a4b22d4342b187ca910a1e6f4ce0ff61b89569/cmd/strelaysrv/status.go#L24). I've no idea if this is documented anywhere (probably not), but it's similar to how it's handles in the main syncthing service (that also has a rest/debug/pprof endpoint you can enable).

Reply:
Mildly feel like this wording is weird, I might say "enable go's builtin profiler" or something, but if it's copy pasted from elsewhere I feel like it's fine to keep it.. probably.